keras_hub/src/models/efficientnet/efficientnet_backbone.py

```diff
@@ -170,12 +172,21 @@ def __init__(
         self._pyramid_outputs = {}
         curr_pyramid_level = 1
 
-        for i in range(len(stackwise_kernel_sizes)):
+        num_stacks = len(stackwise_kernel_sizes)
+
+        if isinstance(depth_coefficient, tuple):
```

Comment:
This is confusing, most that can be passed "stackwise" have `stackwise_` as a prefix, except `depth_coefficient` now.

It's probably better UX to just rename these first arguments `stackwise_depth_coefficient` and `stackwise_width_coefficient`. At least then our argument names are consistent. Remember to update docstrings.

You could keep backwards compat by adding a few lines at the top of the constructor here. Something like this. You have to allow `stackwise_depth_coefficient=None` and `stackwise_width_coefficient=None` in the arg list.

```python
num_stacks = len(stackwise_kernel_sizes)
if "depth_coefficient" in kwargs:
    stackwise_depth_coefficient = kwargs.pop("depth_coefficient") * num_stacks
if "width_coefficient" in kwargs:
    stackwise_width_coefficient = kwargs.pop("width_coefficient") * num_stacks
```

Reply:
done